etc/scdaq/scdaq.conf

```diff
@@ -47,10 +49,10 @@ enable_stream_processor:yes
 # Note: When changing the processing type, change also "output_filename_prefix"
 # in the file output section.
 #
-processor_type:PASS_THROUGH
+processor_type:GMT
```

Comment:
I think this should be commented out in the generic config. For the moment we need to set it to the correct value for the given host and in the future it should come from the FM.

Reply:
agreed with @dinyar

Reply:
done